src/sonic-config-engine/minigraph.py

```diff
@@ -1,4 +1,5 @@
 #!/usr/bin/env python
+from __future__ import print_function
```

Comment:
This line should not be necessary for backward-compatibility with Python 2.7, but it won't hurt anything.

Reply:
I was getting error related to the print function syntax:
```
    Running sonic-cfggen -y /sonic/src/sonic-config-engine/tests/test.yml -t /sonic/src/sonic-config-engine/tests/test.j2
-------------------------------------------------------------------------------------------------------------------------------------------------- Captured stderr call --------------------------------------------------------------------------------------------------------------------------------------------------
Traceback (most recent call last):
  File "/sonic/src/sonic-config-engine/tests/../sonic-cfggen", line 39, in <module>
    from minigraph import minigraph_encoder
  File "/sonic/src/sonic-config-engine/minigraph.py", line 291
    print("VNI must be an integer (use default VNI %d instead)" % vni_default, file=sys.stderr) 
                                                                                   ^
SyntaxError: invalid syntax
```

Reply:
Oh, maybe the `file=` argument was added in Python 3.0.